packages/riipen-ui/src/components/Editor.jsx

```diff
@@ -282,32 +282,35 @@ class Editor extends React.Component {
       }
 
       /* Rich Text Editor */
-      :global .DraftEditor-root {
+      :global(.DraftEditor-root) {
```

Comment:
These were the actual fixes. I think we may have upgraded `styled-jsx` at some point and they changes their interface to this for global styles.